vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,16 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+// Health stores information about the container's healthcheck results.
+type Health struct {
+	Status         string    // States are: "starting", "healthy", "unhealthy"
+	FailingStreak  uint64    // Number of consecutive failures
+	LastCheckStart time.Time // Zero if never started
```

Comment:
Also, may want to include an `Err error` field to keep track of any errors resolving health.